modules/core/src/main/java/org/apache/ignite/IgniteCluster.java

```diff
@@ -527,6 +527,12 @@
      * when all dirty pages are prepared for checkpoint, but not necessarily flushed to disk.
      * <p>
      * WAL state can be changed only for persistent caches.
+     * <p>
+     * <b>NOTE:</b>
+     * Currently, this method should only be called on a stable topology when no nodes are leaving or joining cluster,
```

Comment:
@alamar should we also add todo comment with the reference to the original issue?

Reply:
I don't think this is appropriate place for a TODO. We have this link in developer warning, where it is more convenient in my opinion.

TODOs usually reside in comments which are not javadoc.